### DISCUSSION TITLE
Commercial applications not allowed by terms of service?

### DISCUSSION THREAD

**Discussion by M2S-Carlo** [tidal-music/discussions#113] [General]
Quoting "Version 3.0 of the Tidal Developer Terms of Service" (last updated on May 6, 2024) from the [official documentation guidelines](https://developer.tidal.com/documentation/guidelines/guidelines-developer-terms):

"You may use the Developer Tools for the sole purpose of developing and distributing non-commercial applications, websites or services."

This seems to rule out any meaningful project. Who is going to invest their time and money to support Tidal without any hope of monetization?

Also: what will happen to apps such as Soundiiz, SongShift, TuneMyMusic, etc? Will they be able to continue using the APIs for commercial purposes while new apps won't? This would be unfair and anticompetitive.

Is there anybody from Tidal that can shine some light on this matter?

**Comment by stefanmoro** [comment]
Hey @M2S-Carlo, thanks for the message.

We will work on making the T&C more clear, reflecting that we will also be working with commercial partners who have received explicit approvals for commercial apps.
